charts/tidb-operator/templates/scripts/_create_cert.sh.tpl

```diff
@@ -0,0 +1,159 @@
+set -e
```

Comment:
Does this need to be a .tpl? It seems like it doesn't need to be templated but instead can take in any templating as arguments.

Reply:
> Does this need to be a .tpl? It seems like it doesn't need to be templated but instead can take in any templating as arguments.

Thanks for pointing this.